### PR TITLE
Added new ruby to greet branch

### PR DESCRIPTION
**Pull request by tavisca-gsanap**
I did it by pulling repo to local then resetting to the last good commit and force pushing it again

### CODE REVIEW

File: allDir.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Dir.chdir(".")
```
Comment (tavisca-vtripathi) [inline allDir.rb]: Take user input to look for which directory he/she wants to list

File: allDir.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+subdir_list=Dir["*"].reject{|o| not File.directory?(o)}
+puts subdir_list;
```
Comment (tavisca-vtripathi) [inline allDir.rb]: Wrap it in a method and call that method. Always try to think if your code will then new functionality should not break the old one. It should be backward compatible.

**Review by tavisca-vtripathi — CHANGES_REQUESTED** [review]

**Comment by tavisca-gsanap** [comment]
sir I did the changes as you have mentioned

**Review by tavisca-vtripathi — APPROVED** [review]
Looks good!!

**Pull request merged**